SourceX/dvlnet/tcp_server.h

```diff
@@ -16,6 +16,10 @@ namespace dvl {
 namespace net {
 
 class server_exception : public dvlnet_exception {
+	const char *what() const throw()

```

Comment:
Here it should have `public:` and `override`; class methods in C++ are private by default (unlike structs)